### ISSUE TITLE
Collecting event - host

### ISSUE THREAD

**Issue by OwenLonsdale** [AAFC-BICoE/dina-planning#286]
Data associated with host is in two places - a verbatim section in "Collecting event details" in the "Collecting event" tab, and in "host organism" in a separate region under the "Associations" tab linking to the catalogue of life. 
They should be side by side with a button to automatically transfer verbatim data to the CoL linking field (similar to the structure made for Geographic place). Any "remarks" could be placed underneath this linking pair for fields.

**Comment by cgendreau** [comment]
They do not represent the same thing.

- host organism in Collecting-Event is the host at the collecting-event :)
- host organism on material-sample is usually used when something was grown on something else (initiated and maintained by human) and is not necessarily collected

**Comment by OwenLonsdale** [comment]
Thanks for the clarity.
The overlap in vocabulary confused me, so it will likely confuse others as well. 
So 2 things then: 1) Can the field name be changed to reflect the difference in the two hosts, and 2) can the outside link to host nomenclature be implemented in the first "host"?

**Comment by michellelocke** [comment]
Yes, as @OwenLonsdale said, we still want a way to have Verbatim Host in the Collecting Event and to have a linked host for searchability and a link to the most up-to-date name. It was my understanding that we would use Associations for this and I presumed we would use Host Organism. If this is meant for the living collections then it will need to be removed from the dead collections view, otherwise it WILL be used by people. Perhaps the Field could be change to Living Host Organism (or some way, deemed appropriate by the living collections, to differentiate that it is for living collections). We will also need an additional field in the Collecting Event for an Interpreted Host that is linked to the Taxonomy table.
